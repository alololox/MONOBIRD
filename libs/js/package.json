{
  "name": "root",
  "private": true,
  "devDependencies": {
    "@babel/cli": "^7.7.4",
    "@babel/core": "^7.7.4",
    "@babel/preset-env": "^7.7.4",
    "@babel/preset-react": "^7.7.4",
    "babel-core": "^7.0.0-bridge.0",
    "babel-loader": "^8.0.6",
    "babel-plugin-styled-components": "^1.10.6",
    "lerna": "^3.19.0",
    "webpack": "^4.41.2"
  },
  "scripts": {
    "build": "lerna exec --parallel -- babel src -d lib --ignore **/*.story.js,**/*.spec.js"
  },
  "dependencies": {
    "react": "^16.12.0",
    "react-dom": "^16.12.0",
    "styled-components": "^4.4.1"
  }
}
