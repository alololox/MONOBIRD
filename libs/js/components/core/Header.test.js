import React from "react";
import { render } from "@testing-library/react";
import { Header } from "./index.js";
import "jest-styled-components";

describe("Header Test", () => {
  it("Renders Header", () => {
    const { asFragment } = render(<Header title="Monobird" />);

    expect(asFragment()).toMatchSnapshot();
  });
  it("Renders Fancy Header", () => {
    const { asFragment } = render(<Header fancy={true} title="Fancy" />);

    expect(asFragment()).toMatchSnapshot();
  });
});
