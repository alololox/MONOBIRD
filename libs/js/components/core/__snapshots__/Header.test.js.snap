// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Dirty Little Header Test Renders Header 1`] = `
<DocumentFragment>
  <div
    class="sc-bdVaJa cMNoxm"
  >
    <img
      class="sc-bwzfXH jThacJ"
      src="//images.arcteryx.com/foundation-ui/svgs/ArcBirdWhite.svg"
    />
    <ul
      class="sc-htpNat gtMwsM"
    >
      <li
        class="sc-bxivhb jULzu"
      >
        Monobird
      </li>
      <li
        class="sc-bxivhb jULzu"
      >
        Men's
      </li>
      <li
        class="sc-bxivhb jULzu"
      >
        Women's
      </li>
      <li
        class="sc-bxivhb jULzu"
      >
        Explore
      </li>
    </ul>
  </div>
</DocumentFragment>
`;
